src/object_store/aws.rs

```diff
@@ -171,6 +172,7 @@ impl ObjectStore for S3FileSystem {
 
 #[allow(dead_code)]
 impl S3FileSystem {
+    /// Convenience wrapper for creating a new `S3FileSystem` using default configuration options.  Only works with AWS.
```

Comment:
@seddonm1 do you know if the comment "Only works with AWS" is correct? I havent had time to play with MinIO to know if its true.

Reply:
Your statement is correct as the default endpoint and regions will be set by the AWS Rust SDK